lib/tasks/test.rake

```diff
@@ -3,9 +3,13 @@ require_relative './evm_test_helper'
 if defined?(RSpec)
 namespace :test do
   task :initialize do
-    ENV['RAILS_ENV'] ||= "test"
-    Rails.env = ENV['RAILS_ENV'] if defined?(Rails)
-    ENV['VERBOSE']   ||= "false"
+    if ENV['RAILS_ENV'] && ENV["RAILS_ENV"] != "test"
+      warn "Warning: RAILS_ENV is currently set to '#{ENV["RAILS_ENV"]}'. Forcing to 'test' for this run."
+    end
+    ENV['RAILS_ENV'] = "test"
```

Comment:
Is this line still needed?

Reply:
There are a bunch of other tasks that rely on that value being correct and other things rely on Rails.env, so it's good to set both.